<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customersearch</class>
 <widget class="QDialog" name="customersearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>431</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Customer Details</string>
   </property>
   <widget class="QPushButton" name="pushButton_custsearch">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>47</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_custsearch"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_first"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sur"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_add"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_phone"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
